llvm/include/llvm/TextAPI/RecordsSlice.h

```diff
@@ -134,20 +134,20 @@ class RecordsSlice {
   }
 
   struct BinaryAttrs {
+    std::vector<StringRef> AllowableClients;
+    std::vector<StringRef> RexportedLibraries;
+    std::vector<StringRef> RPaths;
+    StringRef ParentUmbrella;
+    StringRef InstallName;
+    StringRef UUID;
+    StringRef Path;
     FileType fileType = FileType::Invalid;
     llvm::MachO::PackedVersion CurrentVersion;
     llvm::MachO::PackedVersion CompatVersion;
     uint8_t SwiftABI = 0;
```

Comment:
Is SwiftABI still used?

Reply:
I think so, the linker continues to check it. I imagine since abi stability the value is always 7 though.  We chatted about this a while back with @cooperp, but the details are slipping my mind.

Reply:
Oh yeah, the linker still enforces that the Swift ABI matches in all object files.  Thats logic we needed back when changing the ABI happened each release.  Its still there, but just never finds a mismatch.

If the Swift team did ever need to bump this version then the linker will be there to complain if it goes wrong :)

Reply:
Is the check/requirement just object files or also dylibs that link into the final linked image?

Reply:
Looks like dylibs too yeah.  ld64 gets the swift version from used dylibs, tbds and object files, and makes sure they all match